doc/source/ray-dashboard.rst

```diff
@@ -246,31 +247,32 @@ Logical View (Experimental)
 
 **State**: State of an actor.
 
-- 0: Alive
-- 1: Restarting
-- 2: Dead
+- Alive
```

Comment:
Won't we include `Dependencies_UNREADY` here? It looks like I can find them from code.

Reply:
Good catch, I added it thanks.